volontulo/templates/volontulo/show_offer.html

```diff
@@ -3,16 +3,19 @@
 {% block title %}Lista ofert Volontulo{% endblock %}
 
 {% block sidebar %}
-<ul class="list-unstyled">
-    <li><a href="{% url 'show_offer' 1 %}">Oferta 1</a></li>
-    <li><a href="{% url 'show_offer' 2 %}">Oferta 2</a></li>
-    <li><a href="{% url 'show_offer' 3 %}">Oferta 3</a></li>
-    <li><a href="{% url 'show_offer' 4 %}"><strong>Oferta 4</strong></a></li>
-    <li><a href="{% url 'show_offer' 5 %}">Oferta 5</a></li>
-    <li><a href="{% url 'show_offer' 6 %}">Oferta 6</a></li>
-    <li><a href="{% url 'show_offer' 7 %}">Oferta 7</a></li>
-    <li><a href="{% url 'show_offer' 8 %}">Oferta 8</a></li>
-</ul>
+    <ul class="list-unstyled">
+        <li><a href="{% url 'show_offer' 1 %}">Oferta 1</a></li>
+        <li><a href="{% url 'show_offer' 2 %}">Oferta 2</a></li>
+        <li><a href="{% url 'show_offer' 3 %}">Oferta 3</a></li>
+        <li><a href="{% url 'show_offer' 4 %}"><strong>Oferta 4</strong></a></li>
+        <li><a href="{% url 'show_offer' 5 %}">Oferta 5</a></li>
+        <li><a href="{% url 'show_offer' 6 %}">Oferta 6</a></li>
+        <li><a href="{% url 'show_offer' 7 %}">Oferta 7</a></li>
+        <li><a href="{% url 'show_offer' 8 %}">Oferta 8</a></li>
+    </ul>
+    <p>
```

Comment:
correct, and that's why i proposed a div, which is also a block element, just like p, but it doesn't have margins meant for text blocks styling